lmfdb/genus2_curves/isog_class.py

```diff
@@ -106,27 +106,31 @@ def make_class(self):
         self.curves = [ {"label" : c['label'], "equation_formatted" : list_to_min_eqn(c['min_eqn']), "url": url_for_label(c['label'])} for c in curves_data ]
         self.ncurves = curves_data.count()
         self.bad_lfactors = [ [c[0], list_to_factored_poly_otherorder(c[1])] for c in self.bad_lfactors]
-        for endalgtype in ['end_alg', 'rat_end_alg', 'real_end_alg', 'geom_end_alg', 'rat_geom_end_alg', 'real_geom_end_alg']:
+        ## duplication of get_end_alg.  need to clean up!
+        end_alg_title_dict = {'end_ring': '\\mathrm{End}(J)', 
+                              'rat_end_alg': '\\mathrm{End}(J) \\otimes \\Q',
+                              'real_end_alg': '\\mathrm{End}(J) \otimes \R',
```

Comment:
Should this be \\otimes and \\R?

Alternatively, you could use the python construct r'\End(J) \otimes \R' (the r in front lets you single backslash.

We have a macro for End, but I see it needs fixing ... coming in a pull request soon.